<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>219</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>445</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16777215</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>300</height>
    </size>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText</set>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>421</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Interval</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Units</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="cb_units">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Celsius</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fahrenheit</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Current methods:&lt;/p&gt;&lt;p&gt;lm-sensors -- use that package to grab data from third-party application&lt;/p&gt;&lt;p&gt;/proc/acpi -- use standard ACPI thermal zone (may not work on some processors)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Method to use</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="sb_interval">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="sb_overheat_level">
        <property name="toolTip">
         <string>Maximum temperature level before changing to alert color</string>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="cb_method">
        <item>
         <property name="text">
          <string>lm-sensors</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/proc/acpi</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Overheat level</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Colors</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>421</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="KColorButton" name="kcb_color">
        <property name="color">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
        <property name="defaultColor">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Overheat color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="toolTip">
         <string>Default color</string>
        </property>
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="KColorButton" name="kcb_overheat_color">
        <property name="toolTip">
         <string>Alert color</string>
        </property>
        <property name="color">
         <color>
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
